akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -86,7 +86,8 @@ object ReplicatorSettings {
       durableKeys = config.getStringList("durable.keys").asScala.toSet,
       pruningMarkerTimeToLive = config.getDuration("pruning-marker-time-to-live", MILLISECONDS).millis,
       durablePruningMarkerTimeToLive = config.getDuration("durable.pruning-marker-time-to-live", MILLISECONDS).millis,
-      deltaCrdtEnabled = config.getBoolean("delta-crdt.enabled"))
+      deltaCrdtEnabled = config.getBoolean("delta-crdt.enabled"),
+      maxDeltaSize = config.getInt("delta-crdt.max-delta-size"))
```

Comment:
If I am not mistaken this is not a size in bytes but in 'individual deltas'. Maybe good idea to clarify this?